docs/source/responses.rst

```diff
@@ -1,6 +1,63 @@
 API Responses
 =============
 
+A Civis API call from ``client.<endpoint>.<method>`` returns a :class:`civis.response.Response` object
+(or a :class:`civis.response.PaginatedResponse` object, if ``<method>`` is a "list" call):
```

Comment:
Added these notes to document / encourage the use of `.json()` and call out the immutability of the new response objects.

This page in the Sphinx docs is not just for `civis.response.Response`. We'll come back to this page in a future PR to add and organize notes better for other classes and functions mentioned.

Reply:
The doc build has finished for a preview: https://civis-python--487.org.readthedocs.build/en/487/responses.html#api-responses